api/src/test/java/org/pmiops/workbench/profile/ProfileServiceTest.java

```diff
@@ -77,4 +92,203 @@ public void testReturnsLastAcknowledgedTermsOfService() {
     assertThat(profile.getLatestTermsOfServiceVersion()).isEqualTo(1);
     assertThat(profile.getLatestTermsOfServiceTime()).isEqualTo(1);
   }
+
+  @Test
+  public void validateInstitutionalAffiliation() {
+    VerifiedInstitutionalAffiliation affiliation =
+        new VerifiedInstitutionalAffiliation()
+            .institutionShortName("Broad")
+            .institutionDisplayName("The Broad Institute")
+            .institutionalRoleEnum(InstitutionalRole.OTHER)
+            .institutionalRoleOtherText("Kibitzing");
+
+    Profile profile =
+        new Profile()
+            .verifiedInstitutionalAffiliation(affiliation)
+            .contactEmail("kibitz@broadinstitute.org");
+
+    DbInstitution dbInstitution = new DbInstitution();
```

Comment:
I'm usually accepting of a fair amount of repetition in test code, but I think these 4 lines to create a mock Broad institution are used in every single test case. That's my threshold of "bump it into the setup and kill a few dozen lines" in unit tests.